src/frontend/src/routes/router.tsx

```diff
@@ -38,6 +38,7 @@ export const ProtectedRoute = () => {
 
 const Router = () => {
   const isMPO = !!isMPORole();
+  const hideViewComments = false;
```

Comment:
We should be utilizing the statusList to determine if comments are show

Reply:
I may not add this at the root level. I would default the value to true at the function level.